yaml2ncml/templates/roms.tpl

```diff
@@ -0,0 +1,14 @@
+<variable name="grid" type="int">
+        <attribute name="cf_role" value="grid_topology"/>
+        <attribute name="topology_dimension" type="int" value="2"/>
+        <attribute name="node_dimensions" value="xi_psi eta_psi"/>
+        <attribute name="face_dimensions"
+            value="xi_rho: xi_psi (padding: both) eta_rho: eta_psi (padding: both)"/>
+        <attribute name="edge1_dimensions" value="xi_u: xi_psi eta_u: eta_psi (padding: both)"/>
+        <attribute name="edge2_dimensions" value="xi_v: xi_psi (padding: both) eta_v: eta_psi"/>
+        <attribute name="node_coordinates" value="lon_psi lat_psi"/>
+        <attribute name="face_coordinates" value="lon_rho lat_rho"/>
+        <attribute name="edge1_coordinates" value="lon_u lat_u"/>
+        <attribute name="edge2_coordinates" value="lon_v lat_v"/>
+        <attribute name="vertical_dimensions" value="s_rho: s_w (padding: none)"/>
+    </variable>
```

Comment:
@rsignell-usgs I believe that the information here pertains to ROMS only.  However, it is not clear to me what else could be in the template.